crates/simulations/src/cozy/world_contracts.rs

```diff
@@ -1,634 +0,0 @@
-use std::{borrow::Cow, fmt::Debug, sync::Arc};
```

Comment:
No need for `world_contracts` anymore. Agents can directly use the rust bindings to call/execute against the evm with the rust bindings.